index.js

```diff
@@ -42,7 +43,14 @@ function openFile (file, sonic) {
     if (len > 0 && len > sonic.minLength && !sonic.destroyed) {
       actualWrite(sonic)
     }
-  })
+  }
+
+  if (sonic.sync) {
+    const fd = fs.openSync(file, 'a')
+    fileOpened(null, fd)
```

Comment:
I would make sure `'ready'` is emitted on `nextTick` anyway.

Reply:
This will emit `'ready'` twice, one sync which will be lost. Can you please highlight this with a comment?